dummy_robot/dummy_map_server/README.md

```diff
@@ -0,0 +1,96 @@
+## **What Is This?**
+
+This demo creates and runs a ROS2 **node**, `dummy_map_server`, that publishes `nav_msgs::msg::OccupancyGrid` on a ROS2 publisher topic, `/map`.
+
+**dummy_map_server** is also a dependency of **dummy_robot_bringup**. Please refer to [dummy_robot_bringup](https://github.com/ros2/demos/tree/rolling/dummy_robot/dummy_robot_bringup).
+
+## **What Is An Occupancy Grid?**
+
+An occupancy grid is essentially a **2-dimensional** map with each cell containing an integer to indicate the presence, absence or general state of a space in a certain physical room a robot is in. With this set of information, it better informs how a robot can best navigate through obstacles.
+
+Eg.
+
+`-1` represents **unknown** spaces.
+`0` represents **unoccupied** spaces.
+`1` represents **occupied** spaces.
+
+:warning: Note that the integer within a cell is application-specific and can go beyond `-1` and `1`.
+
+![](img/occupancy_grid.png)
+
+The bolded integer indicates the position of a robot with lidar.
+
+## **Build**
+
+```bash
+cd $HOME
+mkdir demo_ws/src
+cd ~/demo_ws/src
+git clone https://github.com/ros2/demos.git
+cd ~/demo_ws
+source /opt/ros/<DISTRO_NAME>/setup.bash
+colcon build --packages-select dummy_map_server
+```
```

Comment:
I'm not sure that all of this works in Windows, since I think the slashes are different. We use tabs in the ROS 2 documents, which use RST. Maybe you could remove most of this section and just keep the colcon command?

Also, I think you want `colcon build --packages-up-to dummy_map_server`.

The same goes for other code blocks.

Reply:
Will attempt conversion to `RST` format then in order to introduce tabs and verify on Windows. This will take a while.

Reply:
Upon further research, the conversion to `RST` format alone will take too long to conclude this pull request. Going for just keeping the colcon command and preserving the current `Markdown` format.